files/zip.go

```diff
@@ -9,16 +9,26 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/amp-labs/cli/logger"
 	"github.com/amp-labs/cli/utils"
 )
 
 var now = time.Now() //nolint:gochecknoglobals
 
+const (
+	TmpDir = ".amp" // This is used for zipping and uploading
+)
+
 func Zip(path string) (zippedFolder string, zipError error) {
 	wd := utils.GetWorkingDir()
+	if wd == "" {
+		logger.Fatal("unable to get working directory")
+
+		return
+	}
 
 	// TODO: create in temporary folder, not cwd
-	dest := filepath.ToSlash(filepath.Join(wd, fmt.Sprintf("amp_%d.zip", now.Unix())))
+	dest := filepath.ToSlash(filepath.Join(wd, TmpDir, fmt.Sprintf("amp_%d.zip", now.Unix())))
```

Comment:
`ToSlash` doesn't make sense here. I know you didn't write this code, but this will break on Windows. I'm not sure what the original intent was (maybe `filepath.Abs`?) but this just seems like a bug waiting to happen.

Reply:
I'm not sure about the intent as well so wary of removing/replacing it. @laurenzlong  might have context

Reply:
Rishabh wrote this code, but I don't believe there was a particular reason for this method, feel free to replace.

Reply:
In that case I'd just get rid of the `filepath.ToSlash` method completely.

```suggestion
	dest := filepath.Join(wd, TmpDir, fmt.Sprintf("amp_%d.zip", now.Unix()))
```

It accomplishes the same thing without breaking cross-platform support.